weekend-bowlling/spec/gameSpec.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+describe ('Game', function() {
+  var game;
+
+  beforeEach(function(){
+      game = new Game();
+  });
+
+    it ('shows the score of first roll', function (){
```

Comment:
Could this have been placed into a nested describe block to split up the behaviour being tested into separate sections?